eng/scripts/docs/Docs-ToC.ps1

```diff
@@ -21,3 +21,200 @@ function Get-NamepspacesFromDll($dllPath) {
     }
     return $namespaces | Sort-Object -Unique
 }
+
+function DownloadNugetPackage($package, $version, $destination) {
+    # $PackageSourceOverride is a global variable provided in
+    # Update-DocsMsPackages.ps1. Its value can set a "customSource" property.
+    # If it is empty then the property is not overridden.
+    $customPackageSource = Get-Variable -Name 'PackageSourceOverride' -ValueOnly -ErrorAction 'Ignore'
```

Comment:
We setup `PackageSourceOverride` variable at `Update Docs Onboarding for Daily branch` step